lib/recurly/risk/three-d-secure/strategy/cybersource.js

```diff
@@ -24,6 +24,10 @@ export default class CybersourceStrategy extends ThreeDSecureStrategy {
       year,
     };
 
+    if(typeof preflightDeviceDataCollector === 'object') {
+      data.billing_info_id = preflightDeviceDataCollector.billingInfoId;
+    }
```

Comment:
Given the above config changes, this can be simplified:

```js
if ('billingInfoId' in preflightDeviceDataCollector) {
  data.billing_info_id = preflightDeviceDataCollector.billingInfoId;
}
```